ingenialink/ethercat/network.py

```diff
@@ -295,6 +295,8 @@ def stop_pdos(self) -> None:
             for servo in self.servos
             if servo.slave.state in [pysoem.OP_STATE, pysoem.SAFEOP_STATE]
         ]
+        if len(op_servo_list) == 0:
```

Comment:
Is this change related?

Reply:
Yes, if no drives are in the OP or SafeOp state (they're in PreOp or disconnected), there is no need to reset them (transition back to PreOp).